src/Spago/Config.js

```diff
@@ -4,8 +4,10 @@ export function updatePackageSetHashInConfigImpl(doc, sha) {
   doc.get("workspace").get("package_set").set("hash", sha);
 }
 
-export function addPackagesToConfigImpl(doc, newPkgs) {
-  const deps = doc.get("package").get("dependencies");
+export function addPackagesToConfigImpl(doc, isTest, newPkgs) {
+  const deps = isTest
+    ? doc.get("package").get("test").get("dependencies")
```

Comment:
The `test` section is optional, so we should also have a test where the package does not have one, and see what happens (we'll presumably have to fix this implementation)